### DISCUSSION TITLE
Skill acquisitions

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#298] [Ideas]
Could we have skills as consumable items, or sellable items in shops? Rather than having a big list of 255 skills swappable by default.

**Comment by MokhaLeee** [comment]
I think it is not a hard thing to introduce skill scroll, maybe we canmake it later

**Reply by JesterWizard** [reply]
Nice job on this. The scrolls work great.

The cherry on top would be having an configuration option that lets the user disable any skills in this list they haven't already obtained. 

![image](https://github.com/user-attachments/assets/79b953da-5da9-40e7-bdfc-fb50926138e8)

**Reply by MokhaLeee** [reply]
`KernelDesigerConfig::debug_autoload_skills`

**Reply by JesterWizard** [reply]
Fantastic!
